thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/db/entity/AnomalyResult.java

```diff
@@ -6,44 +6,25 @@
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
 import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
 
 import org.apache.commons.lang.ObjectUtils;
 import org.joda.time.DateTime;
 
 import com.google.common.base.MoreObjects;
-import com.linkedin.thirdeye.util.ThirdEyeUtils;
 
 @Entity
 @Table(name = "anomaly_results")
 public class AnomalyResult extends AbstractBaseEntity implements Comparable<AnomalyResult> {
 
-  // TODO : User AnomalyFunctionSpec instead of id
-  @Column(name = "function_id", nullable = false)
-  private long functionId;
-
-  @Column(name = "function_type", nullable = false)
-  private String functionType;
-
-  @Column(name = "function_properties", nullable = false)
-  private String functionProperties;
-
-  @Column(name = "collection", nullable = false)
-  private String collection;
-
   @Column(name = "start_time_utc", nullable = false)
   private Long startTimeUtc;
 
   @Column(name = "end_time_utc", nullable = true)
   private Long endTimeUtc;
 
-  @Column(name = "dimensions", nullable = false)
```

Comment:
Need to retain dimensions. Rename it if this is being confused between the dimensions in anomaly functions